### PR TITLE
Update translations.ts with RU lang

### PR DESCRIPTION
**Pull request by ztxone**
Added Russian translation

### CODE REVIEW

**Review by Gennad-r — APPROVED** [review]

File: packages/starlight-cooler-credit/translations.ts [hunk 1]
```diff
@@ -33,5 +33,12 @@ export const Translations = {
         "starlightCoolerCredit.astro.description": "Vuoi creare il tuo sito web statico? →",
         "starlightCoolerCredit.starlight-blog.title": "Creato con Starlight Blog",
         "starlightCoolerCredit.starlight-blog.description": "Vuoi creare il tuo blog Starlight? →",
+    },    
+    ru: {
+        "starlightCoolerCredit.starlight.description": "Хотите создать свою документацию? →",
+        "starlightCoolerCredit.astro.title": "Работает на Astro",
```
Comment (trueberryless) [inline packages/starlight-cooler-credit/translations.ts:39]:
```suggestion
        "starlightCoolerCredit.astro.title": "Создано с помощью Astro",
```

File: packages/starlight-cooler-credit/translations.ts [hunk 2]
```diff
@@ -33,5 +33,12 @@ export const Translations = {
...
+        "starlightCoolerCredit.astro.description": "Хотите создать свой статичный сайт? →",
+        "starlightCoolerCredit.starlight-blog.title": "Работает на блоге Starlight",
```
Comment (trueberryless) [inline packages/starlight-cooler-credit/translations.ts:41]:
```suggestion
        "starlightCoolerCredit.starlight-blog.title": "Создано с помощью Starlight Blog",
```

**Review by trueberryless — COMMENTED** [review]
I have reviewed this PR as a non-native speaker. In general it looks good to me, but I left some suggestions, described here:

As mentioned in the Discord, another translator sent me his version and there are a few differences. I'm not native, so I'm asking you, what you think fits better in those places.

Either just accept the suggestion or ignore them...

File: packages/starlight-cooler-credit/translations.ts (hunk 1 above)
Comment (ztxone) [reply packages/starlight-cooler-credit/translations.ts:39]: In my version it more close to "Works on Astro"
Another one is close to "Created with Astro"

In russian we can say both ways with same meaning. But may be in terms on original words second version ("Создано с помощью Astro") might be more accurate. LGTM

File: packages/starlight-cooler-credit/translations.ts (hunk 2 above)
Comment (ztxone) [reply packages/starlight-cooler-credit/translations.ts:41]: Same here

In my version it means more close to "Works on Starlight Blog"
Another one is close to "Created with Starlight Blog"

Both variants are good for ru-native, but second version more close to original en words. LGTM

**Pull request merged**